front/pages/api/w/[wId]/members/[userId]/index.ts

```diff
@@ -79,28 +79,73 @@ async function handler(
 
   switch (req.method) {
     case "POST":
-      if (
-        !req.body ||
-        !req.body.role ||
-        !["admin", "builder", "user", "revoked"].includes(req.body.role)
-      ) {
-        return apiError(req, res, {
-          status_code: 400,
-          api_error: {
-            type: "invalid_request_error",
-            message: "The request body is invalid, expects { role: string }.",
-          },
+      // TODO(@fontanierh): use DELETE for revoking membership
```

Comment:
Planning to do in the next PR (to avoid making this one too big)

Reply:
ack